src/main/java/uk/gov/ons/ctp/response/action/export/service/ManifestBuilder.java

```diff
@@ -62,6 +62,18 @@ private PrintFileMainfest createManifest(String filename, ByteArrayOutputStream
               "PPD1.1",
               1);
     }
+
+    if (filename.startsWith("P_IC_H2")) {
```

Comment:
maybe we should refactor this so it's just the description that changes, rather than recreating the whole object. Also, use `if...else... else` rather than separate if statements

Reply:
Done